Provide error message when required key constant(s) in `keys.mjs` is missing

howard-e opened the pull request:
[Preview Tests](https://raw.githack.com/w3c/aria-at/bocoup/fix-issue-589/build/index.html)

Address #589.

`npm run build` will now run as expected but will report the issue with a suitable error message.

Eg. `[Command]: The key reference "ENTERR" found in "tests/alert/data/commands.csv" for "test id 1: trigger alert" is invalid. Command may not be defined in "tests/resources/keys.mjs".`

mzgoddard reviewed (changes requested):
The call to `addCommandError` is reporting this error already. What do you think about moving this copy into `addCommandError`? Or we can change `addCommandError` in other ways.

howard-e commented:
> The call to `addCommandError` is reporting this error already. What do you think about moving this copy into `addCommandError`? 

@mzgoddard I went ahead and used the existing `addCommandError` and moved the copy there as suggested. I changed the copy as such: `[Command]: The key reference "ENTERR" found in "tests/alert/data/commands.csv" for "test id 1: trigger alert" is invalid. Command may not be defined in "tests/resources/keys.mjs".`

To address the issue, the `undefined` return for the keypress object (`{ id: ..., keystroke: ... }`) that happens when the key's definition is missing has been defaulted. This means the script will continue running as expected and report the error afterwards as expected.

alflennik reviewed (approved):
Huge improvement in the readability of the errors, nice. It's true the affected test will not run on the review page, but that seems fine given the fact that a command is broken.
